src/main/java/org/broadinstitute/hellbender/utils/read/ReadUtils.java

```diff
@@ -1203,6 +1203,7 @@ public static int getFirstAlignedBaseOffset(final GATKRead read) {
                     result += length;
                 }
             }
+            // reachable if the input read's cigar core-operations (non-clipping ones) consist only of a I operation which is considered valid as per GATKRead#isValid:
```

Comment:
One last comment about this doc: there's no `isValid()` in `GATKRead`, there's one in `SAMRecord`.